src/main/java/com/desiato/puresynth/configurations/AuthTokenFilter.java

```diff
@@ -25,6 +26,9 @@ public AuthTokenFilter(AuthenticationService authenticationService){
         this.authenticationService = authenticationService;
     }
 
+    private static final AntPathMatcher antPathMatcher = new AntPathMatcher();
```

Comment:
Why do you use static variable here? Also instantiating manually is not meeting the practice of dependency injection

Reply:
When I initially introduced AntPathMatcher I read about the fact that it is a utility class with a stateless nature, and so it does not need to be managed or configured per instance.
But now I also read that it is better to inject it for testing purposes, mostly for different AntPathPatcher implementations.